subsystems/provisioning/definitions.go

```diff
@@ -370,6 +376,10 @@ type Config struct {
 
 	// If set, will explicitly enable or disable power save for all wifi connections managed by NetworkManager.
 	WifiPowerSave *bool `json:"wifi_power_save"`
+
+	// If set, will reboot the device after it has been offline for this duration
+	// 0 will disable this feature.
+	DeviceRebootAfterOfflineMinutes int `json:"device_reboot_after_offline_minutes"`
```

Comment:
I consider doing `device_reboot_after_offline_hours` while not super granular. I also can't imagine you would want less than one hour

Reply:
Minutes makes the most sense. All the timeouts should have the same "unit" and minutes feels correct. Also, with the way Timeout parsing works, you can use decimal units if needed. Or "1h"... take a look at the Timeout.Unmarshall() function. It treats bare numbers as seconds currently, but we can change that to minutes, but people can specify easier units if they want. "10h30m" and such (we don't have to document/advertise that extra feature!)